strata/cmd/mongo/lreplica_drivers/lrs3driver/lrs3driver.go

```diff
@@ -27,10 +27,12 @@ type AWSOptions struct {
 
 // ReplicaOptions are used for commands like backup and restore
 type ReplicaOptions struct {
-	MaxBackgroundCopies int    `long:"max-background-copies" default:"16" description:"Backup and restore actions will use up to this many goroutines to copy files"`
-	Port                int    `long:"port" default:"27017" description:"Backup should look for a mongod instance that is listening on this port"`
-	Username            string `long:"username" description:"If auth is configured, specify the username with admin privileges here"`
-	Password            string `long:"password" description:"Password for the specified user."`
+	DatabaseHostname            string `long:"database-hostname" default:"localhost" description:"Database hostname can be override with a specific hostname in most cases localhost is sufficient"`
```

Comment:
I think a name like `LocalHostname` would be better than `DatabaseHostname`. 

Do you think the following would description would make sense? `'localhost' or a hostname that is accessible on the local machine via e.g. kubernetes network`. 

By the way, another reason to avoid the name `DatabaseHostname` is that the MongoDB database might be distributed over multiple hosts.

Reply:
Sorry, I have totally forgotten this PR, because I changed the company and moved to a new city. :(
I've implemented your requested changes.